### DISCUSSION TITLE
Contribution: Interest in Adding Devcontainer Configuration?

### DISCUSSION THREAD

**Discussion by CodeMan99** [Orange-OpenSource/hurl#2539] [Ideas] [answered]
Hello!

Wondering if you'd be interested in a contribution of some devcontainer configuration? This project's contributing guide is very good. I believe devcontainers are a fantastic way to reduce the barrier to entry.

In this case specifically, I would make sure to use the latest version of the rust toolchain. The base image could be the Debian Bookworm variant of the [rust devcontainer image](https://github.com/devcontainers/images/blob/main/src/rust/history/1.0.9.md). Of course the dockerfile would install the suggested [debian packages](https://github.com/Orange-OpenSource/hurl?tab=readme-ov-file#debian-based-distributions). Then add in features for [CircleCI CLI](https://github.com/CodeMan99/features/tree/main/src/circleci-cli), [GitHub CLI](https://github.com/devcontainers/features/tree/main/src/github-cli), & [Python](https://github.com/devcontainers/features/tree/main/src/python).

Happy to answer any questions or provide more documentation!

**Comment by jcamiel** [comment, marked as answer]
Hi @CodeMan99 

I confess I don't really know what's a devcontainer and wouldn't be at ease to maintain it. I would prefer to improve our current documentation at <https://hurl.dev> / [contributing guide](https://github.com/Orange-OpenSource/hurl/blob/master/CONTRIBUTING.md) / [integration guide](https://github.com/Orange-OpenSource/hurl/blob/master/integration/README.md) if things are not clear enough to get started.

That said, nothing prevents you for doing a repo with a devcontainer for Hurl!

Thanks

**Reply by CodeMan99** [reply]
The existing documentation is great. Just looking to make some of those docs executable. :)

It's totally fair if you're not interested in maintaining the configuration.
